DemoQA/Pages/userPage.js

```diff
@@ -0,0 +1,44 @@
+import webTablesPage from "./webTablesPage";
+import { Selector, t } from "testcafe";
+
+
+export default class UserInfoRow {
+    constructor(index) {
+        this.container = webTablesPage.rowLine.nth(index);
+        this.editBttn = this.container.find("[title~='Edit']");
+        this.deleteBttn = this.container.find("[title~='Delete']");
+        this.firstName = this.container.find(".rt-td:nth-child(1)");
+        this.lastName = this.container.find(".rt-td:nth-child(2)");
+        this.ageNo = this.container.find(".rt-td:nth-child(3)");
+        this.emailAddress = this.container.find(".rt-td:nth-child(4)");
+        this.salary = this.container.find(".rt-td:nth-child(5)")
+        this.department = this.container.find(".rt-td:nth-child(6)");
+        this.firstNameInput = Selector("#firstName");
```

Comment:
There are no input fields on the UserInfoRow, so feel free to delete these Selectors.